podkrepi-bg/frontend
# bugfix/1949-Semiannual-Report-2022-Typo

Replaced Financial_Semiannual_Report_2022.pdf file with the updated one without the typos

Closes [#1949](https://github.com/podkrepi-bg/frontend/issues/1949)

## Screenshots:

![image](https://github.com/user-attachments/assets/91dff757-b3a6-4114-9d20-7f8e3cb5974d)



## Testing

### Steps to test

1. Open https://podkrepi.bg/finance-reports/Podkrepi.bg_Financial_Semiannual_Report_2022.pdf
2. Scroll to page 3 of the pdf document
3. Ensure typo in SoftUni is corrected

### Affected urls







## Environment

### New environment variables:




### New or updated dependencies:



Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|
